driver.py

```diff
@@ -0,0 +1,301 @@
+    print("StackStatus ", currentStackStatus)
+    while currentStackStatus != 'CREATE_COMPLETE':
+        monitorStackCreationStatus()
+    sys.exit()
+
+def monitorStackDestroyStatus():
+    currentStackResp = getStackStatus()
+    if currentStackResp is not None and currentStackResp.stdout is not None and currentStackResp.returncode == 0:
+        getSecretResponseJson = json.loads(currentStackResp.stdout.decode("utf-8"))
+        currentStackStatus = getSecretResponseJson['Stacks'][0]["StackStatus"] 
+        print("StackStatus ", currentStackStatus)
+        while currentStackStatus != 'DELETE_COMPLETE':
+            monitorStackDestroyStatus()
+    sys.exit()
+
+
+
+def destroyStack():
+    '''
+    aws cloudformation delete-stack --stack-name apigw-lambda-bedrock
+    '''
+    deleteStackResponse = subprocess.run(
+        [
+            "aws",
+            "cloudformation",
+            "delete-stack",
+            "--stack-name",
+            "apigw-lambda-bedrock"
+        ],
+        capture_output=True,
+    )
+    handleOperation(operationName='Destroy Stack',response=deleteStackResponse)
+
+
+def deleteBucket():
+    '''
+    aws s3 rb s3://your-bucket-name --force
+    '''
+    deleteBucketResponse = subprocess.run(
+        [
+            "aws",
+            "s3",
+            "rb",
+            "s3://logs-bucket-test-1",
+            "--force"
+        ],
+        capture_output=True,
+    )
+    handleOperation(operationName='Delete Bucket',response=deleteBucketResponse)
+
+def createBucket():
+    '''
+    aws s3api create-bucket --bucket logs-bucket-test-1
+    '''
+    createBucketResponse = subprocess.run(
+        [
+            "aws",
+            "s3api",
+            "create-bucket",
+            "--bucket",
+            "logs-bucket-test-1"
+        ],
+        capture_output=True,
+    )
+    handleOperation(operationName='Create Bucket',response=createBucketResponse)
+
+
+def loadingAnimation(process) :
+    while process.is_alive() :
+        chars = "/—\|" 
+        for char in chars:
+            sys.stdout.write('\r'+'.. '+char)
+            time.sleep(.1)
+            sys.stdout.flush()
+
+
+
+def getApiGateWayEndpoint():
+    api_name="HttpApi4328"
+    stage_name="dev"
+
+    getApiGwEndpointResponse = subprocess.run(
+        [
+            "aws",
+            "apigatewayv2",
+            "get-apis",
+            "--query",
+            "Items[?Name=='"+api_name+"'].ApiEndpoint",
+            "--output",
+            "text"
+        ],
+        capture_output=True,
+        text=True
+    )
+    gateWayUrl = getApiGwEndpointResponse.stdout.removesuffix("\n")
+    if gateWayUrl == '':
+        print("Stack creation is still in progress , please execute the command after few seconds")
+    else:
+        invoke_url=gateWayUrl + "/" + stage_name
+        print("API gateway url that can be invoked : ",invoke_url)
+
+if __name__ == '__main__':
+    printBanner(banner="Lynx Lab")
```

Comment:
can we also please add this ASCII art after/before the name -

("`-''-/").___..--''"`-._ 
 `6_ 6  )   `-.  (     ).`-.__.`) 
 (_Y_.)'  ._   )  `._ `. ``-..-' 
   _..`--'_..-_/  /--'_.'
  ((((.-''  ((((.'  (((.-' 
  
  courtesy : asciiart - https://www.asciiart.eu/animals/cats